homeassistant/components/skybell/config_flow.py

```diff
@@ -17,6 +18,40 @@
 class SkybellFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Skybell."""
 
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self._email: str
+
```

Comment:
The type hint should be defined outside `__init__` routine, maybe name it `_reauth_email`:
```python
class SkybellFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
    """Handle a config flow for Skybell."""

    _reauth_email: str
```